client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -409,22 +409,24 @@ const Editor = (props: EditorProps): JSX.Element => {
       return accessibility;
     };
 
-    const setTabTrapped = (trapped: boolean) => {
+    const setTabTrapped = (trapped: boolean, announce = true) => {
       setMonacoTabTrapped(trapped);
       store.set('monacoTabTrapped', trapped);
-      ariaAlert(
-        `${
-          trapped
-            ? t('learn.editor-alerts.tab-trapped')
-            : t('learn.editor-alerts.tab-free')
-        }`
-      );
+      if (announce) {
+        ariaAlert(
+          `${
+            trapped
+              ? t('learn.editor-alerts.tab-trapped')
+              : t('learn.editor-alerts.tab-free')
+          }`
+        );
+      }
     };
 
     // By default, Tab will be trapped in the monaco editor, so we only need to
     // check if the user has turned this off.
     if (!isTabTrapped()) {
-      setTabTrapped(false);
+      setTabTrapped(false, false);
```

Comment:
```suggestion
      setTabTrapped(false, { announce: false });
```
This reads a little nicer, I think.